text/0000-test-directories.md

```diff
@@ -0,0 +1,56 @@
+- Start Date: 2020-01=09
+- Relevant Team(s): Ember.js
+- RFC PR: https://github.com/emberjs/rfcs/pull/575
+- Tracking:
+
+# Test Directories
+
+## Summary
+
+This RFC proposes that we change the names of an Ember app's test directories to reflect the names
+in the guides. For example, instead of `tests/acceptance/*-test.js`, we should have
+`tests/application/*-test.js`.
+
+## Motivation
+
+Renaming test directories to match the Guides will make it easier to have a shared language
+between tests, and to reduce confusion for newer users when using Ember CLI.
+
+## Detailed design
+
+These are the directory transformations that need to happen:
+
+- tests/acceptance -> tests/application
+- tests/integration > tests/rendering
+- tests/unit/utils -> tests/unit
+- tests/unit/controllers -> tests/container/controllers
```

Comment:
Do we have a reason these are called container now? I think unit is generally a good way to look at testing this at the interface level.

Reply:
Not sure where that comes from. I scanned #268 and #119 but they don't seem to be the place where the terminology is introduced. Maybe @rwjblue knows?

Reply:
From [his comments above](https://github.com/emberjs/rfcs/pull/575#pullrequestreview-355235832), it does seem like #268 is where they came up:

> However, I strongly dislike "container" as a name here

> I regret a bit the names we landed on in #268, and perhaps this is good time to discuss those names.

Reply:
Looking through #268 I can't seem to find the introduction of it in the doc or comments.